kotlin-runtime/scaffolding/pom.xml

```diff
@@ -185,7 +204,7 @@
                                 ${project.build.directory}/dependency/ftl-runtime-${ftl.version}.jar
                             </plugin>
                         </plugins>
-                        <input>${project.basedir}/src/main/kotlin,${project.build.directory}/generated-sources/ftl</input>
+                        <input>${project.basedir}</input>
```

Comment:
unfortunately had to change this back because it fails when the input directory is empty, which is the case for the generated-sources input when no other modules are deployed to the cluster